mytk/tests/testLabel.py

```diff
@@ -17,8 +17,9 @@ def setUp(self):
     def tearDown(self):
         self.app.quit()
 
-    def start_timed_mainloop(self, function, timeout=500):
-        self.app.root.after(int(timeout/4), function)
+    def start_timed_mainloop(self, function=None, timeout=500):
+        if function is not None:
+            self.app.root.after( int(timeout/4), function)    
```

Comment:
👸 type of comment, but :
```suggestion
            self.app.root.after(int(timeout/4), function)
```